designsafe/static/scripts/ng-designsafe/services/user-service.js

```diff
@@ -15,7 +15,7 @@ export function UserService($http, $q) {
     };
 
     this.getPublic = function (username) {
-      return $http.get("/api/users/public", {params: {username: username}})
+      return $http.get("/api/users/public/", {params: {username: JSON.stringify(username)}})
```

Comment:
should we change references here to `usernames` instead of `username` if we're always assuming an array now?

Reply:
Yes that makes sense. I'll swap it out.